cmd/crowdsec-cli/alerts.go

```diff
@@ -533,7 +533,7 @@ func (cli *cliAlerts) NewInspectCmd() *cobra.Command {
 		DisableAutoGenTag: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 0 {
-				printHelp(cmd)
+				_ = cmd.Help()
```

Comment:
is there a reason to underscore the return value when you can just use the function directly?

Reply:
yes, there are linters that complain about unchecked errors. we don't use them for now but it's a nice to have